baseapp/abci_utils.go

```diff
@@ -230,6 +230,11 @@ func (h *DefaultProposalHandler) SetTxSelector(ts TxSelector) {
 	h.txSelector = ts
 }
 
+// SetSignerExtractionAdapter sets the SetSignerExtractionAdapter on the DefaultProposalHandler.
+func (h *DefaultProposalHandler) SetSignerExtractionAdapter(signerExtAdapter mempool.SignerExtractionAdapter) {
+	h.signerExtAdapter = signerExtAdapter
```

Comment:
we use like [this](https://github.com/MANTRA-Chain/mantrachain/pull/381/files) actually

Reply:
How are you getting transactions to pass through here?

https://github.com/cosmos/cosmos-sdk/blob/3f8ef3689a27840db628aa146069e2e71039d8cb/baseapp/abci_utils.go#L301-L306